crates/sncast/src/response/print.rs

```diff
@@ -76,6 +63,7 @@ impl From<Value> for OutputValue {
             ),
             Value::String(s) => OutputValue::String(s.to_string()),
             Value::Bool(b) => OutputValue::String(b.to_string()),
+            Value::Number(n) => OutputValue::String(n.to_string()),
```

Comment:
Why do we need this?

Reply:
It's a result of deleting `Decimal`, here https://github.com/foundry-rs/starknet-foundry/pull/3392/files#diff-b07be4e12cdea6b79c88ef81620caabdbf5d4250c3f73c912fc0eda9467b3bbeL11